HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,20 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: name
+Returns the curent server name
+code::
+//
+~myInternalServer=Server.internal;
+~myInternalServer.boot.reboot
+~myInternalServer.name
+
+//
+~myLocalServer=Server.local;
+~myLocalServer.boot.reboot
+~myLocalServer.name
+::
+
```

Comment:
Please use single-letter variables for the code examples—this matches the style of the rest of the documentation. This applies to the other PR's (#2579, #2581, #2582, #2583). The code example here could be reduced to:
```
Server.local.name;
Server.internal.name;
```
since servers do not need to be booted to have a name.